org.pegasus_frontend.Pegasus.yml

```diff
@@ -0,0 +1,28 @@
+app-id: org.pegasus_frontend.Pegasus
+runtime: org.kde.Platform
+runtime-version: '5.12'
```

Comment:
Any reason for not using 5.13?

Reply:
5.13 dropped the Canvas3D module, but its alternative, Qt3D has been [found to be broken](https://bugreports.qt.io/browse/QTBUG-66587) in certain platforms in the past. At the moment Canvas3D support is still assumed, but I can upgrade to later Qt versions in the future.